src/routes.tsx

```diff
@@ -151,7 +168,19 @@ export const reactPageRoutes: ReactPageRoute[] = [
     exact: true,
   },
   {
-    path: LOCALE_PATH_PATTERN + '/new-member/:name?/:id?',
+    path: LOCALE_PATH_PATTERN + '/new-member',
+    render: ({ match }: RouteComponentProps<any>) => (
+      <EmbarkRoot
+        language={match.params.locale}
+        name={undefined}
+        baseUrl={undefined}
+        showLanding={true}
+      />
```

Comment:
Maybe we could just render `<Landing language={match.params.locale} />` here instead? Then we could actually remove the `showLanding` prop and everything related to that from `EmbarkRoot`. Would make it easier to understand what's going on IMO. But I might be missing something @gustaveen ?

It works fine locally for me, the only thing is of course that with the `<Redirect to={`/${match.params.locale}/new-member`} />` we get like a new page load (with white screen) instead of a smooth transition to the landing page. But I think it's worth it, seeing that the routing here gets more declarative.

Reply:
Hmm, wouldn't expect it to behave like that. `<Redirect />` should only perform a client side navigation, no?

I agree that this could be taken a step further but didn't want to change too many things before you could have a look!

I'm also realizing I don't need to pass `undefined` to the props - could just leave them out.

Reply:
Did some investigation. There are a few changes that need to be made to make it work. However, I think separating `Landing` from `EmbarkRoot` would simplify things. Would recommend doing it as part of a separate PR though.

Reply:
Yeah, I was confused by the white screen. But `<Redirect />` does replace the current location in the history stack though - but that should still not be the "real" history stack if you get what I mean 🙃 Or? 
But anyways, I don't think it matters since it's just if you mess with the url after already being in a flow and I don't know why anyone would do that. Truly an edge case

Reply:
Okey, cool I think I understand! I thought about adding the "push" prop to add the redirect to history but seemed better just to replace (since that endpoint doesn't exist).

Could it be because of the fade animation (opacity 0 -> 100)?

I made a draft of the changes needed to extract Landing from EmbarkRoot here btw: https://github.com/HedvigInsurance/web-onboarding/pull/523/files?diff=split&w=1

Reply:
No I agree, it makes more sense to just replace the location 👌 

Also, nice draft PR - I really think we should do it that way! 🙌  It's so confusing that the landing page, which really has nothing to do with Embark, is baked into the `EmbarkRoot` component

Reply:
And yeah, maybe it's because of the animation 🧠 Then I guess we could just add the same black background to the `Landing` component so it's black before the stuff fades in? Then it would look the same as when it's rendered in `EmbarkRoot`

Reply:
Yeah that would be a smart thing to add indeed! Will look into it

Reply:
Body is already black by default so then I don't think we can do much about the "white flash". Perhaps it only happens in local dev though...